### ISSUE TITLE
Label Integration container as default container

### ISSUE THREAD

**Issue by astefanutti** [apache/camel-k#2200]
When the Integration Pods are composed of more than one containers, like with Service Mesh, it's useful to instruct commands like `logs` or `exec` to default to the `integration` container, where Camel routes run.

A new label, `kubectl.kubernetes.io/default-container`, has been introduced to meet that need, so it'd be useful to have it set to `integration` on the Integration Deployment, KnativeService, or CronJob, depending on the deployment strategy.

**Comment by lakshayletsgo** [comment]
Hey @astefanutti is this issue still open? I want to work on it.

**Comment by gansheer** [comment]
Hi @lakshayletsgo, it looks like nothing has been done in Camel-K on that side, and the [KEP](https://github.com/kubernetes/enhancements/blob/master/keps/sig-cli/2227-kubectl-default-container/README.md) has been implemented so it probably is.

@squakez WDYT ?

**Comment by squakez** [comment]
Yes, this is one is still relevant and we're happy to receive any contribution. No one else is actively working on it afaik, so, @lakshayletsgo feel free to work on it and submit a PR.
